cmd/jaeger/internal/extension/jaegerstorage/extension.go

```diff
@@ -176,7 +176,7 @@ func (s *storageExt) Start(_ context.Context, host component.Host) error {
 			grpcTelset := telset
 			grpcTelset.Metrics = scopedMetricsFactory(storageName, "grpc", "tracestore")
 			//nolint: contextcheck
```

Comment:
How does OTEL Exporter (similar principle of using gRPC client) deal with context in this scenario? At minimum I think we should pass it to the constructor, but more specifically my question is: should the constructor do something with it, like pass it when creating a connection?

Reply:
the change lgtm